float32[3] pos_sp
uint8[3] pos_halt
uint8 enable
uint8 mode
